src/map/packet_system.cpp

```diff
@@ -3257,18 +3257,32 @@ void SmallPacket0x050(map_session_data_t* const PSession, CCharEntity* const PCh
     uint8 equipSlotID = data.ref<uint8>(0x05); // charequip slot
     uint8 containerID = data.ref<uint8>(0x06); // container id
 
-    if (containerID != LOC_INVENTORY && containerID != LOC_WARDROBE && containerID != LOC_WARDROBE2 && containerID != LOC_WARDROBE3 &&
-        containerID != LOC_WARDROBE4 && containerID != LOC_WARDROBE5 && containerID != LOC_WARDROBE6 && containerID != LOC_WARDROBE7 &&
-        containerID != LOC_WARDROBE8)
+    bool isAdditionalContainer =
+        settings::get<bool>("main.EQUIP_FROM_OTHER_CONTAINERS") &&
```

Comment:
you goofed, the setting still should have been in isEquippableInventory

Reply:
Its equivalent, no? 

- If non of the regular containers
-If setting is enabled, and one of the other other containers

Reply:
no, because now these are not additional containers unless setting is true.

Reply:
that was the intention: You shouldn't be able to equip directly from these containers (using addons) unless the setting is true (default is false)

Reply:
o you aren't understanding this isn't about if you can equip from them lol